transcription/email.go

```diff
@@ -0,0 +1,30 @@
+package transcription
+
+import (
+	"log"
+	"net/smtp"
+	"os"
+	"strings"
+)
+
+func sendEmail(from string, to []string, subject string, body string) {
+	// Set up authentication information.
+	auth := smtp.PlainAuth(
+		"",
+		os.Getenv("MAIL_EMAIL"),
+		os.Getenv("MAIL_PASSWORD"),
+		"smtp.gmail.com",
+	)
```

Comment:
There is no character limit convention in Go. How come you don't want to put this all on the same line? Do you think this is more readable?